### DISCUSSION TITLE
Import "googleapiclient.discovery" could not be resolved

### DISCUSSION THREAD

**Discussion by eleondida** [googleapis/google-api-python-client#2329] [Q&A]
Hello, 

I'm trying to get started with the YouTube Data API vs in Python but I'm getting: from googleapiclient.discovery import build
ModuleNotFoundError: No module named 'googleapiclient'.

google-api-python-client is correctly installed

How can I resolve this issue?

**Comment by K0-RR** [comment]
I also have this issue, but ONLY on Windows. I tried Fedora and openSUSE - the same script works fine there...

**Reply by michaelhayesnyc** [reply]
I have the same issue, extremely frustrating. Can someone please come to the rescue?

**Comment by JohannesK71083** [comment]
The problem is that you need specific versions. For example the following requirements.txt for pip does work:
```
google-api-python-client==1.7.2
google-auth==1.8.0
google-auth-httplib2==0.0.3
google-auth-oauthlib==0.4.1
```
